utils/polus-imagej-util/{{cookiecutter.project_slug}}/src/__init__.py

```diff
@@ -0,0 +1,5 @@
+from pathlib import Path
+import sys
+
+ij_converter_path = Path(__file__).parents[1].joinpath('src')
+sys.path.append(str(ij_converter_path))
```

Comment:
It's unclear to me why this is needed.

Reply:
Required for unit testing plugin from in external directory.

Reply:
Removed `__init__.py ` and modified system path variables in `unit_test.py`